docs/content/en/docs-dev/user-guide/managing-piped/configuring-notifications.md

```diff
@@ -83,6 +83,23 @@ spec:
           mentionedAccounts:
             - '@user1'
             - 'user2'
+      - name: integration-slack-api-with-oauthTokenData-and-mentioned-groups
+        slack:
+          oauthTokenData: token
+          channelID: testid
+          mentionedGroups:
+            - 'group1'
+            - '<!subteam^group2>'
+      - name: integration-slack-api-with-oauthTokenData-and-mentioned-both-accounts-and-groups
+        slack:
+          oauthTokenData: token
+          channelID: testid
+          mentionedAccounts:
+            - 'user1'
+            - '@user2'
+          mentionedGroups:
+            - 'group1'
+            - '<!subteam^group2>'
```

Comment:
Using the groupID or teamID in the documents makes it easier for users to use this feature.

```suggestion
            - 'groupID1'
            - '<!subteam^groupID2>'
```